### PR TITLE
Peer review history manager ui

### PR DESCRIPTION
**Pull request by RakshandaWankhede**
Done -peer review history

### CODE REVIEW

File: client/src/constants.js [hunk 1]
```diff
@@ -139,3 +139,34 @@ export const countryData = [
     ]
   }
 ]
+export const quarterInfo = [
```
Comment (bhushan-oe) [inline client/src/constants.js:142]: why do we need this? Can you think of other logic? anything at server side?

**Review by bhushan-oe — CHANGES_REQUESTED** [review]

File: client/src/constants.js (hunk 1 above)
Comment (RakshandaWankhede) [reply client/src/constants.js:142]: Done at server side

File: server/app/api/controllers/peer_review.js [hunk 2]
```diff
@@ -92,12 +92,36 @@ module.exports = {
       });
   },
   getForManager: function (req, res, next) {
-    const { functional_manager, startDate, endDate, selectedYear } = req.query
+    const { functional_manager, selectedYear, value } = req.query
+    let startDate, endDate
+    switch (value) {
+      case 'Quarter 1':
+        startDate = new Date(`2 January ${selectedYear}`);
```
Comment (bhushan-oe) [inline server/app/api/controllers/peer_review.js]: why 2nd Jan? and not the 1st?

File: server/app/api/controllers/peer_review.js [hunk 3]
```diff
@@ -92,12 +92,36 @@ module.exports = {
...
+        endDate = new Date(`1 April ${selectedYear}`)
```
Comment (bhushan-oe) [inline server/app/api/controllers/peer_review.js]: it should be 31st march

**Review by bhushan-oe — CHANGES_REQUESTED** [review]

File: server/app/api/controllers/peer_review.js (hunk 2 above)
Comment (RakshandaWankhede) [reply server/app/api/controllers/peer_review.js]: modified

File: server/app/api/controllers/peer_review.js (hunk 3 above)
Comment (RakshandaWankhede) [reply server/app/api/controllers/peer_review.js]: modified

**Comment by RakshandaWankhede** [comment]
same files available in PR #46 closing this PR

**Pull request closed without merging**